tests/unit/test_loader.py

```diff
@@ -5,6 +5,7 @@
     Test Salt's loader
 """
 
+
```

Comment:
We should also move these to pytest, not necessarily a requirement for **this** PR

Reply:
Yes we should eventually do this for sure. :D